### PR TITLE
wikirunner/cloneRepository

### PR DESCRIPTION
**Pull request by MarcelDiessner**
#175

### CODE REVIEW

File: runners/wikiConsole/templates/cloneRepository.asciidoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Clone Repository
+=============
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/cloneRepository.asciidoc:2]: This line is too short, so the header is not displayed as a header line

File: runners/wikiConsole/templates/cloneRepository.asciidoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Prerequisites
+~~~~~~~~~~~~~~
+* Install git
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/cloneRepository.asciidoc:6]: You should mention here what to do to install Git

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: runners/wikiConsole/templates/cloneRepository.asciidoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Clone Repository
+================
+
+Prerequisites
+~~~~~~~~~~~~~
+* You need to have git installed. You can download git https://git-scm.com/downloads[here] for your OS and install it by following the instructions.
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/cloneRepository.asciidoc:6]: Git is a name, so the first letter has to be always capital

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: runners/wikiConsole/templates/cloneRepository.asciidoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+Download
+~~~~~~~~
+Open a terminal in your current workspace.
+<% if (directoryPath) { %>
+Firstly, you have to move to the directory where the repository should be cloned to.
+If the directory does not exist yet, you can create by using the `mkdir` command in the terminal.
```
Comment (denise-khuu) [inline runners/wikiConsole/templates/cloneRepository.asciidoc:13]: *create it

**Review by GuentherJulian — APPROVED** [review]

**Review by denise-khuu — APPROVED** [review]

**Pull request merged**